sduino/hardware/sduino/stm8/libraries/I2C/I2C.c

```diff
@@ -167,15 +177,15 @@ uint8_t sendByte(uint8_t i2cData) {
   return 0;
 }
 
-uint8_t I2C_scan(uint8_t i2cAddress) {
-  ret = start();
-  stopIfNotSuccess(ret, 1);
+// uint8_t I2C_scan(uint8_t i2cAddress) {
```

Comment:
I tried to get this part to work, but somehow if there is an error the i2c code get's stuck. I ordered a protocol analyser to debug this